PowerPointLabs/PowerPointLabs/ActionFramework/NarrationsLab/AddNarrations/AddNarrationsActionHandler.cs

```diff
@@ -29,8 +36,15 @@ protected override void ExecuteAction(string ribbonId)
                 NotesToAudio.IsRemoveAudioEnabled = true;
                 this.GetRibbonUi().RefreshRibbonControl("RemoveNarrationsButton");
             }
-
-            List<string[]> allAudioFiles = NotesToAudio.EmbedSelectedSlideNotes();
+            List<string[]> allAudioFiles = new List<string[]>();
+            try
+            {
+                allAudioFiles = NotesToAudio.EmbedSelectedSlideNotes();
+            }
+            catch
+            {
+                return;
```

Comment:
Will there be some exception added here later?

Reply:
If ExecuteAction (part of ActionFramework) cannot be executed properly, the code should show an error message to the user. This seems to just return and do nothing,

Reply:
After catching the exception, the code should read the exception message and show it to the user?

Reply:
Edited according to suggestion!